commission/wizards/commission_make_settle.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 Quartile
+# Copyright 2014-2022 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from datetime import date, timedelta
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, fields, models
+
+
+class CommissionMakeSettle(models.TransientModel):
+    _name = "commission.make.settle"
+    _description = "Wizard for settling commissions"
+
+    date_to = fields.Date("Up to", required=True, default=fields.Date.today())
+    agent_ids = fields.Many2many(
+        comodel_name="res.partner", domain="[('agent', '=', True)]"
+    )
+    settlement_type = fields.Selection(selection=[], required=True)
+    can_settle = fields.Boolean(
+        compute="_compute_can_settle",
+        help="Technical field for improving UX when no extra *commission is installed.",
+    )
+
+    def _compute_can_settle(self):
```

Comment:
```suggestion
    @api.depends("date_to")
    def _compute_can_settle(self):
```
Wasn't totally sure if this suggestion would be valid, but compute doesn't seem to run without a trigger.  Here is an image from runboat.
![image](https://user-images.githubusercontent.com/7766939/203697965-e557b305-905b-49dc-8d79-c8ee47045315.png)